src/modeles/mesures.js

```diff
@@ -42,7 +42,7 @@ class Mesures extends InformationsHomologation {
 
   parStatut() {
     const mesuresGeneralesParStatut = this.mesuresGenerales.parStatut();
-    return this.mesuresSpecifiques.parStatut(mesuresGeneralesParStatut);
+    return this.mesuresSpecifiques.parStatutEtCategorie(mesuresGeneralesParStatut);
   }
```

Comment:
Que penses-tu de renommer également la méthode englobante ?

Et donc… peut-être également renommer `this.mesuresGenerales.parStatut();`.
Et remonter jusqu'à `homologation.mesuresParStatut`.

Autre remarque : je verrais bien 2 commits – un pour le renommage, et un autre pour ajouter le `|| !mesure.categorie`.

Et peut-être que renommer autant de choses ne vaut pas le coup (je vois les variables dans les tests, les mocks) … et qu'on peut simplement ajouter le `|| ! mesure.categorie`